core/types/calls.cc

```diff
@@ -560,7 +571,7 @@ DispatchResult dispatchCallSymbol(const GlobalState &gs, const DispatchArgs &arg
                 // suggest adding `extend T::Helpers`.
                 if (args.name == core::Names::declareInterface() || args.name == core::Names::declareAbstract() ||
                     args.name == core::Names::declareFinal() || args.name == core::Names::declareSealed() ||
-                    args.name == core::Names::mixesInClassMethods()) {
+                    args.name == core::Names::mixesInClassMethods() || args.name == core::Names::requiresAncestor()) {
```

Comment:
Just so we can suggest include `T::Helpers`

Reply:
```suggestion
                    args.name == core::Names::mixesInClassMethods() || (args.name == core::Names::requiresAncestor() && gs.requiresAncestorEnabled)) {
```